src/main/java/hudson/plugins/msbuild/MsBuildBuilder.java

```diff
@@ -304,10 +306,12 @@ public Descriptor<Builder> getDescriptor() {
      * @param args
      * @return
      */
+
+    @Nonnull
```

Comment:
Second half of fixing my mistaken reference.
```suggestion
    @NonNull
```